packages/vscode-apollo/README.md

```diff
@@ -1,18 +1,154 @@
-- Displays performance statistics from [Apollo Engine](https://www.apollographql.com/docs/engine/) inline with your queries
-- Jump-to-definition for fragments and schema types
-- Detects fragment references and shows them next to definitions
+- Instant feedback and [intelligent autocomplete](#autocomplete) for fields, arguments, types, and variables as you write queries
+- Seamlessly manage your client side schema alongside your remote one
+- [See performance information](#performance-insights) inline with your query definitions
+- Loads GraphQL schemas and queries automatically from an Apollo Config file
+- Adds [syntax highlighting](#syntax) for GraphQL files and gql templates inside JavaScript files
+- Detects and loads client-side schemas and validates client side field usage in operations
+- [Jump-to-definition](#jump-to-def) for fragments and schema types
+- Supports [local](#local-schemas) and [client-only](#client-only-schemas) schemas
+- Allows [switching schema tags](#commands) to work on upcoming features
+- And more...
 
-## Configuration
+<img src="./images/marketplace/jump-to-def.gif" width="80%" style="margin: 5%" alt="Using jump to definition on a fragment">
 
-Please refer to the [Editor Plugin](https://www.apollographql.com/docs/platform/editor-plugins.html) docs for configuration instructions.
+<h2 id="getting-started">Getting Started</h2>
 
-## Issues
+To get started, first **install the Apollo GraphQL extension**. After installation, GraphQL syntax highlighting should automatically be enabled for `.graphql`, `.gql`, `.js` and `.ts` file types.
 
-Please report any issues you find to the [Apollo Tooling](https://github.com/apollographql/apollo-tooling) repo. Pull requests are always welcome!
+<h3 id="linking-a-schema">Linking a schema</h2>
+
+To get all of the benefits of the VS Code experience, its best to link the schema that is being developed against. The best way to do that is by [publishing a schema](./schema-registry.html#publish) to the Apollo schema registry. Once that is done, two steps are needed:
+
+1. Create an `apollo.config.js` at the root of the project
+2. Copy an API key from the Engine dashboard of the published service
+
+<h3 id="apollo-config">Setting up an Apollo config</h3>
+In order for the VS Code plugin to know how to find the schema, it needs to be linked to either a published schema or a local one. To link a project to a published schema, edit the `apollo.config.js` file to look like this:
+
+```js
+module.exports = {
+  client: {
+    service: "my-graphql-app"
+  }
+};
+```
+
+The service name is the id of the service created in Engine and can be found in the services dashboard of [Engine](https://engine.apollographql.com)
+
+> Important note: If the name of the service in Engine is changed, this value should be the service id. This can be found in the url when browsing the service in Engine. This will be easier to manage in the near future
```

Comment:
```suggestion
> Important note: If the name of the service in Engine is changed, this value should be the service id. This can be found in the url when browsing the service in Engine. This will be easier to manage in the near future.
```

Reply:
I think if we leave this last sentence in, it'll probably sit there for a long time. I'd just leave it out.